module/Activity/config/module.config.php

```diff
@@ -68,6 +68,28 @@
                             ]
                         ]
                     ],
+                    'archive' => [
+                        'type' => 'Literal',
+                        'options' => [
+                            'route' => '/archive',
+                            'defaults' => [
+                                'action' => 'archive'
+                            ]
+                        ]
+                    ],
+                    // Route for categorizing albums by association year.
```

Comment:
Albums? Could you fix this comment or remove it to prevent confusion in the future.